docs/sql-migration-guide.md

```diff
@@ -254,6 +254,8 @@ license: |
         </tr>
     </table>
 
+    - Since Spark 3.0, the unary arithmetic operator plus(`+`) only accepts string, numeric and interval type values as inputs. Besides, `+` with a integral string representation will be coerced to double value, e.g. `+'1'` results `1.0`. In Spark version 2.4 and earlier,  there is no type checking for it, thus, all type values with a `+` prefix are valid, e.g. `+ array(1, 2)` is valid and results `[1, 2]`. Besides, there is no implicit type coercion for `+` with string, e.g. in Spark 2.4, the result of `+'1'` is string `1`.
```

Comment:
nit: the indentation is wrong. It should be 2 spaces.

Reply:
BTW it's worth mention that, in 2.4 the unary plus operator is ignored. There is no type coercion at all, not only for string type.